{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"error"},{"kind":"text","text":": Swift"},{"kind":"text","text":"."},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["iOS"]}]},{"kind":"content","content":[{"anchor":"discussion","level":2,"type":"heading","text":"Discussion"},{"style":"note","content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"Currently is only used for "},{"type":"reference","isActive":true,"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/NetworkLogger\/DecodingError"},{"type":"text","text":"."}]}],"type":"aside","name":"Note"}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/pulse\/networklogger\/responseerror\/error"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.github.pulse\/documentation\/Pulse\/NetworkLogger\/ResponseError\/error","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Contains the underlying error."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"error"},{"kind":"text","text":": Swift"},{"kind":"text","text":"."},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"?"}],"title":"error","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"s:5Pulse13NetworkLoggerC13ResponseErrorV5errors0E0_pSgvp","modules":[{"name":"Pulse"}]},"hierarchy":{"paths":[["doc:\/\/com.github.pulse\/documentation\/Pulse","doc:\/\/com.github.pulse\/documentation\/Pulse\/NetworkLogger","doc:\/\/com.github.pulse\/documentation\/Pulse\/NetworkLogger\/ResponseError"]]},"references":{"doc://com.github.pulse/documentation/Pulse/NetworkLogger/ResponseError/error":{"role":"symbol","title":"error","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"error"},{"kind":"text","text":": Swift"},{"kind":"text","text":"."},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"?"}],"abstract":[{"type":"text","text":"Contains the underlying error."}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/NetworkLogger\/ResponseError\/error","kind":"symbol","type":"topic","url":"\/documentation\/pulse\/networklogger\/responseerror\/error"},"doc://com.github.pulse/documentation/Pulse/NetworkLogger/ResponseError":{"role":"symbol","title":"NetworkLogger.ResponseError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ResponseError"}],"abstract":[],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/NetworkLogger\/ResponseError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ResponseError"}],"url":"\/documentation\/pulse\/networklogger\/responseerror"},"doc://com.github.pulse/documentation/Pulse":{"role":"collection","title":"Pulse","abstract":[{"type":"text","text":"Logger and network inspector for Apple platforms."}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse","kind":"symbol","type":"topic","url":"\/documentation\/pulse"},"doc://com.github.pulse/documentation/Pulse/LoggerStore":{"role":"symbol","title":"LoggerStore","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"LoggerStore"}],"abstract":[{"type":"text","text":"Persistently stores logs, network requests, and response blobs."}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/LoggerStore","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"LoggerStore"}],"url":"\/documentation\/pulse\/loggerstore"},"doc://com.github.pulse/documentation/Pulse/NetworkLogger":{"role":"symbol","title":"NetworkLogger","fragments":[{"kind":"keyword","text":"class"},{"kind":"text","text":" "},{"kind":"identifier","text":"NetworkLogger"}],"abstract":[{"type":"text","text":"A wrapper on top of "},{"type":"reference","isActive":true,"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/LoggerStore"},{"type":"text","text":" that simplifies logging of network requests."}],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/NetworkLogger","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"NetworkLogger"}],"url":"\/documentation\/pulse\/networklogger"},"doc://com.github.pulse/documentation/Pulse/NetworkLogger/DecodingError":{"role":"symbol","title":"NetworkLogger.DecodingError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"DecodingError"}],"abstract":[],"identifier":"doc:\/\/com.github.pulse\/documentation\/Pulse\/NetworkLogger\/DecodingError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"DecodingError"}],"url":"\/documentation\/pulse\/networklogger\/decodingerror"}}}